rpc_jobs/rpc_octavia_gating.yml

```diff
@@ -0,0 +1,46 @@
+ - project:
+    name: "rpc-octavia"
+
+    # This is the URL for the project's repository
+    # which contains the scripts which will be
+    # executed.
+    repo_url: "https://github.com/rcbops/rpc-octavia"
+
+    # This is the whitelist for branch names which
+    # should trigger the job automatically. As this
+    # job only implements PR tests, each item is used
+    # as a Java regex match.
+    branches:
+      - "master"
+
+    # This is a list of names to designate the image
+    # used when preparing a cloud instance. Each name
+    # must also include the key IMAGE with a value
+    # that corresponds to the appropriate image UUID
+    # or name of the image available in the cloud.
+    image:
+      - xenial:
+          IMAGE: "Ubuntu 16.04 LTS (Xenial Xerus) (PVHVM)"
```

Comment:
What flavor would you need to do your tests on - the default is performance1-1 which will not be able to run RPC-O. If you want the same as RPC-O you'll need to use the following immediately under the IMAGE.

```
FLAVOR: "performance2-15"
```

Also, if you're planning on executing RPC-O installs, then you should be using this value for IMAGE: ``Ubuntu 16.04.2 LTS prepared for RPC deployment``

Reply:
+1